src/core/TooltipTable/index.tsx

```diff
@@ -22,15 +22,18 @@ const TooltipTableContent = (props: TooltipTableContentProps): JSX.Element => {
   return (
     <TableContainer {...rest}>
       {contentAlert && <Alert>{contentAlert}</Alert>}
-      {data?.map((section) => (
-        <Section disabled={section.disabled} key={`${section.label}`}>
+      {data?.map((section, index) => (
+        <Section
+          disabled={section.disabled}
+          key={`${section.label?.concat(index.toString())}`}
```

Comment:
super nit: Maybe this? Personally seems more readable 😄 
```suggestion
          key={section.label + String(index)}
```

Reply:
Also prefer `String(foo)` over `foo.toString()`, so when `foo` is `undefined` or `null`, it won't throw a runtime TypeError and will just render string `undefined ` and `null` instead!

Reply:
Woow ,Thanks for the suggestion. I just refactored it 🥇 
Much better now :D